Specifications/Language/README.md

```diff
@@ -1,6 +1,6 @@
 # Q# Language
 
-Q# is part of Microsoft's [Quantum Development Kit](http://www.microsoft.com/quantum), and comes with rich IDE support and tools for program visualization and analysis.
+Q# is part of Microsoft's [Quantum Development Kit](https://www.microsoft.com/quantum), and comes with rich IDE support and tools for program visualization and analysis.
```

Comment:
Some things like http-vs-https we could check with a markdownlint pass on this repo directly; thoughts, @bettinaheim?

Reply:
Sure. Would that generate a warning, or be a failing check on a PR? Does that pass do anything else?